shared-module/displayio/EPaperDisplay.h

```diff
@@ -56,7 +56,7 @@ typedef struct {
     bool black_bits_inverted;
     bool color_bits_inverted;
     bool refreshing;
-    bool always_toggle_chip_select;
+    display_chip_select_behavior_t always_toggle_chip_select;
```

Comment:
Change name of `always_toggle_chip_select` now to just `chip_select` or `chip_select_behavior`.